dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/JSF23CDIGeneralTests.java

```diff
@@ -107,6 +108,8 @@ public void testPostRenderViewEvent() throws Exception {
             // Construct the URL for the test
             URL url = JSFUtils.createHttpUrl(jsf23CDIServer, contextRoot, "");
 
+           jsf23CDIServer.waitForStringInLog("CWWKZ0001I: Application " + contextRoot + " started");
```

Comment:
I don't think you need to add this - the app that's used for this test - `JSFApplication` - invokes `server.addInstalledAppForValidation()`, which should this case a bit more robustly

Reply:
i had trouble without that line before, but things seem to work even with it removed removed